assets/js/components/product-preview/index.js

```diff
@@ -13,9 +13,16 @@ import './style.scss';
  * Display a preview for a given product.
  */
 const ProductPreview = ( { product } ) => {
+	const { placeholderImgSrc } = window.wcSettings;
+
 	let image = null;
 	if ( product.images.length ) {
 		image = <img src={ product.images[ 0 ].src } alt="" />;
+	} else {
+		image = <img
+			src={ placeholderImgSrc }
+			alt={ __( 'Placeholder Image', 'woo-gutenberg-products-block' ) }
```

Comment:
This alt text should be an empty string– a placeholder image is just visual decoration, so we don't need to add information for screen readers.